t/benchmark/multi_consumer_work.lua

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env tarantool
+
+local clock = require('clock')
+local os = require('os')
+local fiber = require('fiber')
+local queue = require('queue')
+
+-- Set the number of consumers.
+local consumers_count = 100
+-- Set the number of tasks processed by one consumer per iteration.
+local batch_size = 100
+
+local barrier = fiber.cond()
+local wait_count = 0
+
+box.cfg()
+
+local test_queue = queue.create_tube('test_queue', 'fifo', {temporary = true})
+
+local function prepare_consumers()
+    local consumers = {}
+
+    for i = 1, consumers_count do
+        consumers[i] = fiber.new(function()
```

Comment:
There is no significant difference between fiber.new and fiber.create. To be compatible with Tarantool 1.9 we could use fiber.create. "new" was introduced only in 1.10. But up to you.

Reply:
I don't mind (in this case). Updated.